bridges/snowbridge/primitives/outbound-queue/src/v2/message_receipt.rs

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+use crate::Log;
+use alloy_core::{primitives::B256, sol, sol_types::SolEvent};
+use codec::Decode;
+use frame_support::pallet_prelude::{Encode, TypeInfo};
+use sp_core::{RuntimeDebug, H160};
+use sp_std::prelude::*;
+
+sol! {
+	event InboundMessageDispatched(uint64 indexed nonce, bool success, bytes32 reward_address);
+}
+
+/// Envelope of the delivery proof
+#[derive(Clone, RuntimeDebug)]
+pub struct MessageReceipt<AccountId>
```

Comment:
@yrong please rename for me.

```suggestion
pub struct DeliveryReceipt<AccountId>
```

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/6/commits/b92cfec4bf0e994443658c0aa9d6c99ab35755fb